erpnext/stock/doctype/packing_slip/test_packing_slip.py

```diff
@@ -1,11 +1,8 @@
 # Copyright (c) 2015, Frappe Technologies Pvt. Ltd. and Contributors
 # See license.txt
 
-import unittest
-
-# test_records = frappe.get_test_records('Packing Slip')
 from frappe.tests.utils import FrappeTestCase
 
 
-class TestPackingSlip(unittest.TestCase):
+class TestPackingSlip(FrappeTestCase):
```

Comment:
I'm not sure if testing will be required for packing slip... Maybe UI testing for barcode scan validation.

Reply:
I suppose it could be good to add some tests to ensure qty is correct with multiple packing slips against a delivery note.